src/main/java/com/pi4j/drivers/display/text/hd44780/Hd44780Driver.java

```diff
@@ -0,0 +1,302 @@
+package com.pi4j.drivers.display.text.hd44780;
+
+import com.pi4j.io.OnOffWrite;
+import com.pi4j.io.i2c.I2C;
+import com.pi4j.drivers.io.expander.pcf8574.Pcf8574OutputDriver;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
```

Comment:
this is driver code for a HD44780 with both parallel and I2C connections? maybe this needs some more (JavaDoc) explanation

Reply:
Added